tools/lsp/ui/api.slint

```diff
@@ -121,11 +121,36 @@ export enum BrushKind {
     radial,
 }
 
+export enum WidgetMode { edit, preview }
```

Comment:
I think none of this belongs int api.slint.

This file is the interface between business logic and the UI and none of this seem relevant for the business logic. You would have changed rust code if it was.

Can we move this elsewhere, please?